test/conversion.jl

```diff
@@ -1,7 +1,8 @@
 @testset "conversion from GAP" begin
 
     ## Defaults
-    @test GAP.gap_to_julia(GAP.GAPInputType, true )
```

Comment:
Using this internal, non-exported type in a test also seems strange -- no user should ever have used `GAPInputType` like that.